lms/djangoapps/discussion/django_comment_client/utils.py

```diff
@@ -691,10 +691,9 @@ def permalink(content):
     else:
         course_id = content['course_id']
     if content['type'] == 'thread':
-        return reverse('discussion.views.single_thread',
-                       args=[course_id, content['commentable_id'], content['id']])
+        return reverse('single_thread', args=[course_id, content['commentable_id'], content['id']])
```

Comment:
The url name changed? And was not updated here?

Reply:
In old version of Django we could access that using a string, but after the upgrade either you need to import the django view here or give url-name. I have provided url name. 
https://docs.djangoproject.com/en/3.0/ref/urlresolvers/